Library/List/SubscriptionAddOnList.cs

```diff
@@ -100,13 +102,25 @@ public void Add(AddOn planAddOn, int quantity, int unitAmountInCents)
         // sub.AddOns.Add(code); 1, unitInCents=this.Plan.UnitAmountInCents[this.Currency]
         public void Add(string planAddOnCode, int quantity=1)
         {
+            if (_subscription == null)
+            {
+                throw new ValidationException(
```

Comment:
If you do want to use the Plan AddOn object instead of just specifying planAddOnCode, quantity, and unitAmountInCents, you need to have a subscription or else you'll run into errors trying to determine the currency.

Reply:
I just realized this particular one doesn't actually take in an AddOn object, my above comment was inaccurate. But still, it can't just know what currency to use, so you need a `_subscription` for this to work